igvc_gazebo/launch/igvc_control.launch

```diff
@@ -10,6 +10,13 @@
     args="$(find igvc_gazebo)/config/igvc_joint_control.yaml"/>
 
   <node name="igvc_control" pkg="igvc_gazebo" type="control">
+      <!-- PID values for wheel speed -->
+      <param name="speed_P_left" value="0.5"/>
+      <param name="speed_P_right" value="0.5"/>
+      <param name="speed_D_left" value="3.0"/>
+      <param name="speed_D_right" value="3.0"/>
```

Comment:
No I for PID?

Reply:
we didn't used to use it on the actual robot before, so it wasn't implemented.